release/ReleasePipeline

```diff
@@ -25,7 +25,9 @@ pipeline {
         }
         stage('Create Images') {
             steps {
-                sh 'release/images.sh'
+                catchError(buildResult: 'SUCCESS', stageResult: 'FAILURE') {
+                    sh 'release/images.sh'
+                }
```

Comment:
+1, I like this.  It accurately captures how images are an optional part of our build process.

Reply:
This is up to discussion. But being optional or not does not imply that this change is part of generalizing the finalization of scripts, therefore it needs to be moved to its own commit (preferable list the rational for the change as well).